src/components/Floorplan.tsx

```diff
@@ -68,60 +138,76 @@ export const ClickableFloorplan: React.FC<ClickableFloorplanProps> = ({
-          const pulseSize =
-            pulseMinSize +
-            ((Math.sin(Date.now() * 0.001 + index) + 1) / 2) *
-              (pulseMaxSize - pulseMinSize);
+          // const pulseMaxSize = 20; // Increased from 8
+          // const pulseMinSize = 10; // New minimum size
+          // const pulseSize =
+          //   pulseMinSize +
+          //   ((Math.sin(Date.now() * 0.001 + index) + 1) / 2) *
+          //     (pulseMaxSize - pulseMinSize);
 
           // Draw outer pulse
-          ctx.beginPath();
-          ctx.arc(point.x, point.y, pulseSize, 0, 2 * Math.PI);
-          ctx.fillStyle = point.isDisabled
-            ? `rgba(75, 85, 99, ${0.4 - ((pulseSize - pulseMinSize) / (pulseMaxSize - pulseMinSize)) * 0.3})`
-            : `rgba(59, 130, 246, ${0.4 - ((pulseSize - pulseMinSize) / (pulseMaxSize - pulseMinSize)) * 0.3})`;
-          ctx.fill();
-
-          // Draw the main point
-          ctx.beginPath();
-          ctx.arc(point.x, point.y, 6, 0, 2 * Math.PI);
-          ctx.fillStyle = gradient;
-          ctx.fill();
-
-          // Draw a white border
-          ctx.strokeStyle = "white";
-          ctx.lineWidth = 2;
-          ctx.stroke();
+          // ctx.beginPath();
+          // ctx.arc(point.x, point.y, pulseSize, 0, 2 * Math.PI);
+          // ctx.fillStyle = point.isDisabled
+          //   ? `rgba(75, 85, 99, ${0.4 - ((pulseSize - pulseMinSize) / (pulseMaxSize - pulseMinSize)) * 0.3})`
+          //   : `rgba(59, 130, 246, ${0.4 - ((pulseSize - pulseMinSize) / (pulseMaxSize - pulseMinSize)) * 0.3})`;
+          // ctx.fill();
 
           if (point.wifiData) {
             const wifiInfo = point.wifiData;
-            const frequencyBand = wifiInfo.channel > 14 ? "5 GHz" : "2.4 GHz";
-            const apLabel =
-              apMapping.find((ap) => ap.macAddress === wifiInfo.bssid)
-                ?.apName ?? wifiInfo.bssid;
-            const annotation = `${frequencyBand}\n${apLabel}`;
+            const iperfInfo = point.iperfResults;
+            const frequencyBand = wifiInfo.channel > 14 ? "5GHz" : "2.4GHz";
+            // const apLabel =
+            //   apMapping.find((ap) => ap.macAddress === wifiInfo.bssid)
+            //     ?.apName ?? wifiInfo.bssid + " " + wifiInfo.rssi;
+            // const annotation = `${frequencyBand}\n${apLabel}`;
+
+            // Draw the main point
+            ctx.beginPath();
+            ctx.arc(point.x, point.y, 8, 0, 2 * Math.PI);
+            ctx.fillStyle = point.isDisabled
+              ? "rgba(156, 163, 175, 0.9)"
+              : getGradientColor(rssiToPercentage(wifiInfo.rssi));
+            ctx.fill();
+
+            // Draw a white border
+            ctx.strokeStyle = "grey";
+            ctx.lineWidth = 2;
+            ctx.stroke();
+            // const t = getGradientColor(rssiToPercentage(wifiInfo.rssi));
+
+            const annotation = `${rssiToPercentage(wifiInfo.rssi)}%`;
+            // These are no longer displayed
```

Comment:
why not btw? I did consider speed as an important metric for getting the wifi quality. Again, could be maybe less verbose, I agree.